src/compass/utils/h5_helpers.py

```diff
@@ -320,129 +524,161 @@ def poly1d_to_h5(group, poly1d_name, poly1d):
+            Meta('tau_0', eap.tau_0, 'slant range time', {'units': 'seconds'}),
+            Meta('tau_sub', eap.tau_sub, 'slant range time',
+                 {'units': 'seconds'}),
+            Meta('theta_sub', eap.theta_sub, 'elevation angle',
+                 {'units': 'radians'}),
+            Meta('ascending_node_time',
+                 eap.ascending_node_time.strftime(TIME_STR_FMT),
+                 'Ascending node crossing time (ANX)',
+                 {'format': 'YYYY-MM-DD HH:MM:SS.6f'})
+        ]
+        eap_group = correction_group.require_group('elevation_antenna_pattern')
+        for meta_item in eap_items:
+            add_dataset_and_attrs(eap_group, meta_item)
+
+    # write out calibration metadata, if present
+    if burst.burst_calibration is not None:
+        cal = burst.burst_calibration
+        cal_items = [
+            Meta('basename', cal.basename_cads, ''),
+            Meta('azimuth_time', cal.azimuth_time.strftime(TIME_STR_FMT),
+                 'Start time', {'format': 'YYYY-MM-DD HH:MM:SS.6f'}),
+            Meta('line', cal.line, 'line'),
+            Meta('pixel', cal.pixel, 'pixel'),
+            Meta('sigma_naught', cal.sigma_naught, 'sigma_naught'),
+            Meta('beta_naught', cal.beta_naught, 'beta_naught'),
+            Meta('gamma', cal.gamma, 'gamma'),
+            Meta('dn', cal.dn, 'dn'),
+        ]
+        cal_group = correction_group.require_group('calibration')
+        for meta_item in cal_items:
+            add_dataset_and_attrs(cal_group, meta_item)
+
+    # write out noise metadata, if present
+    if burst.burst_noise is not None:
+        noise = burst.burst_noise
+        noise_items = [
+            Meta('basename', noise.basename_nads, ''),
+            Meta('range_azimith_time',
+                 noise.range_azimith_time.strftime(TIME_STR_FMT),
+                 'Start time', {'format': 'YYYY-MM-DD HH:MM:SS.6f'}),
+            Meta('range_line', noise.range_line, 'Range line'),
+            Meta('range_pixel', noise.range_pixel, 'Range pixel array'),
+            Meta('range_lut', noise.range_lut, 'Range lookup table data'),
+            Meta('azimuth_first_azimuth_line', noise.azimuth_first_azimuth_line,
+                 'azimuth_first_azimuth_line'),
+            Meta('azimuth_first_range_sample', noise.azimuth_first_range_sample,
+                 'azimuth_first_range_sample'),
+            Meta('azimuth_last_azimuth_line', noise.azimuth_last_azimuth_line,
+                 'azimuth_last_azimuth_line'),
+            Meta('azimuth_last_range_sample', noise.azimuth_last_range_sample,
+                 'azimuth_last_range_sample'),
+            Meta('azimuth_line', noise.azimuth_line, 'azimuth_line'),
+            Meta('azimuth_lut', noise.azimuth_lut, 'azimuth_lut')
+        ]
+        noise_group = correction_group.require_group('noise')
+        for meta_item in noise_items:
+            add_dataset_and_attrs(noise_group, meta_item)
```

Comment:
@seongsujeong Can you please take a look at the description strings for these items? The descriptions I've used are insufficient. Thanks.